frontend/public/components/utils/units.js

```diff
@@ -147,6 +147,78 @@ units.dehumanize = (value, typeName) => {
   return convertValueWithUnitsToBaseValue(value, type.units, type.divisor);
 };
 
+const humanizeUsageUnit = (unit, type, singular) => {
+  switch (type) {
+    case 'memory':
+    case 'limits.memory':
+    case 'requests.memory':
+    case 'storage':
+      if (!unit) {
+        return unit;
+      }
+      return `${unit }B`;
+    case 'cpu':
+    case 'limits.cpu':
+    case 'requests.cpu': {
+      if (unit === 'm') {
+        unit = 'milli';
+      }
+      const suffix = (singular) ? 'core' : 'cores';
+      return (unit || '') + suffix;
+    }
+    default:
+      return unit;
+  }
+};
+
+const usageAmountAndUnit = (value, type, humanizeUnits) => {
```

Comment:
You shouldn't need this. Use `units.dehumanize`.

Reply:
fixed